### PR TITLE
#185475162 Users should be able to order multiple items at once

### PR DESCRIPTION
**Pull request by nishimweprince**
## What does this PR do?

This PR allows a user to checkout multiple orders at once.

### Why
A user needs to be able to order items in bulk or multiple orders at the same time.

### Acceptance Criteria

User should be able to submit multiple orders at once without using a loop, and the backend should handle them in bulk.

## How should this be manually tested?

- Clone the repository using `git clone https://github.com/atlp-rwanda/e-comm-team-techsmith-fn`

- Checkout to the branch using `git checkout ft-multiple-orders-checkout-185475162`

- Install node packages that the application depends on: `npm i`

- Create a `.env` file in the root directory and add all environment variables required to run the application. The required variables are listed in the `.env.example` file available in root directory of the application.

- Start the application development server: `npm start`

- Check out the components documentation: `npm run storybook`

## Any background context you want to provide?

- This PR also improves the Seller 2FA by providing them with short 6-digit codes instead of lengthy tokens for validation.
-  By default, users can order the last three orders that they created.

## What are the relevant pivotal tracker/Trello stories?

- [#185475162](https://www.pivotaltracker.com/story/show/185475162)

## Screenshots (if appropriate)

Screenshot 2023-06-25 at 02 52 23
Screenshot 2023-06-25 at 11 39 58
Screenshot 2023-06-25 at 11 40 09
Screenshot 2023-06-25 at 11 41 41

### CODE REVIEW

File: src/App.jsx
Comment (irakevine) [inline src/App.jsx:1]: It's great that you have protected all these routes
you did well

File: src/components/Pagination.jsx
Comment (irakevine) [inline src/components/Pagination.jsx:1]: Better to add pagination

File: src/containers/HomeContainer.jsx
Comment (irakevine) [inline src/containers/HomeContainer.jsx:1]: Here it is good that the user can have a way to see how all the orders, 
it is very good to remember

File: src/containers/MultipleOrdersContainer.jsx
Comment (irakevine) [inline src/containers/MultipleOrdersContainer.jsx:1]: It's much better to get used to using scss than using regular css

File: src/states/api/apiSlice.js
Comment (irakevine) [inline src/states/api/apiSlice.js:1]: It's also good that you know how to use this alternative method and it doesn't damage anything, still works well.
very good indeed

**Review by irakevine — APPROVED** [review]
Great work!

**Review by niyontwali — APPROVED** [review]

**Review by frankflej — APPROVED** [review]
Great work!

**Review by jcharles921 — APPROVED** [review]
Great job

**Review by ReneNtabana — APPROVED** [review]
Nicely done

File: src/containers/MultipleOrdersContainer.jsx
Comment (nishimweprince) [reply src/containers/MultipleOrdersContainer.jsx:1]: It's not css actually I was chaining ternary operators to handle different scenarios (payment for both a single order and bulk orders) 👍🏽

**Review by Dyves1 — APPROVED** [review]

File: src/states/api/apiSlice.js
Comment (nishimweprince) [reply src/states/api/apiSlice.js:1]: Thanks

**Pull request merged**